### PR TITLE
Added x64 configuration options and common build properties to .NET code

### PR DESCRIPTION
**Pull request by tmajest**
Add common build properties file for .NET code and add x64 configuration options.

Jira: [REEF-120](https://issues.apache.org/jira/browse/REEF-120)

### CODE REVIEW

File: lang/cs/Source/build.props [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (jwang98052) [inline lang/cs/Source/build.props]: Add license header to it.

**Comment by markusweimer** [comment]
Please add a commit message as defined in the contributor's guide :-)

File: lang/cs/ReefDotNet.sln [hunk 2]
```diff
@@ -1,8 +1,6 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.21005.1
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio 2012
```
Comment (markusweimer) [inline lang/cs/ReefDotNet.sln]: Why are we moving to VS 2012?

File: lang/cs/Source/REEF/reef-applications/CLRBridgeClient/CLRBridgeClient.csproj [hunk 3]
```diff
@@ -1,22 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (markusweimer) [inline lang/cs/Source/REEF/reef-applications/CLRBridgeClient/CLRBridgeClient.csproj]: Please keep the license header.

File: lang/cs/Source/REEF/reef-applications/Evaluator/Evaluator.csproj [hunk 4]
```diff
@@ -1,22 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<!--
```
Comment (markusweimer) [inline lang/cs/Source/REEF/reef-applications/Evaluator/Evaluator.csproj]: Please keep the license header.

File: lang/cs/Source/REEF/reef-common/ReefCommon/ReefCommon.csproj [hunk 5]
```diff
@@ -1,22 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<!--
```
Comment (markusweimer) [inline lang/cs/Source/REEF/reef-common/ReefCommon/ReefCommon.csproj]: Please keep the license header.

File: lang/cs/Source/REEF/reef-common/ReefDriver/ReefDriver.csproj [hunk 6]
```diff
@@ -1,22 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<!--
```
Comment (markusweimer) [inline lang/cs/Source/REEF/reef-common/ReefDriver/ReefDriver.csproj]: Please keep the license header.

File: lang/cs/Source/REEF/reef-examples/HelloCLRBridge/HelloCLRBridge.csproj [hunk 7]
```diff
@@ -1,22 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<!--
```
Comment (markusweimer) [inline lang/cs/Source/REEF/reef-examples/HelloCLRBridge/HelloCLRBridge.csproj]: Please keep the license header.

File: lang/cs/Source/REEF/reef-examples/RetainedEvalCLRBridge/RetainedEvalCLRBridge.csproj [hunk 8]
```diff
@@ -1,22 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<!--
```
Comment (markusweimer) [inline lang/cs/Source/REEF/reef-examples/RetainedEvalCLRBridge/RetainedEvalCLRBridge.csproj]: Please keep the license header.

**Comment by markusweimer** [comment]
All the license headers on the .csproj files disappeared. Where they removed by VS?

**Comment by tmajest** [comment]
I accidentally left them out when I was working on them.  I'll add them back and update the .sln file to use VS 2013

**Comment by tmajest** [comment]
I've addressed these comments in the latest commit.

File: lang/cs/ReefDotNet.sln [hunk 9]
```diff
@@ -54,106 +53,140 @@ Global
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{79E7F89A-1DFB-45E1-8D43-D71A954AEB98}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```
Comment (yingdachen) [inline lang/cs/ReefDotNet.sln]: do we still need all these AnyCpu settings?

Comment (jwang98052) [inline lang/cs/ReefDotNet.sln]: We are getting rid of CBT build. Moving to Apache, we should be able to support both x64 and  any CPU, right?

**Pull request closed without merging**